_posts/2021-05-05-the-competitive-advantage-of-ux.md

```diff
@@ -0,0 +1,78 @@
+
+We need to bring UX into the conversation because "as technology products have become more commoditised, UX has become a competitive differentiator", ([Talia Lavie](https://techbeacon.com/devops/8-ways-integrate-ux-speed-devops)) and we know that long term competitive advantage is not met by commoditisation that focuses only on technical details as the differentiator. It is the usability of the product or service that offers long term competitive advantage ([Alex Castrounis](https://www.innoarchitech.com/blog/why-product-user-experience-design-simplicity-drive-competitive-advantage#:~:text=It's%20all%20about%20competitive%20advantage,leadership%20as%20we'll%20discuss)). For example, [Jared Spool ](https://articles.uie.com/gaining-a-competitive-advantage-with-a-solid-experience-vision/) explains competitive advantage comes from understanding the competition's customers. Such insight "could drive the competitor’s customers to switch to our product or service if we built something that truly embodied their needs".
+
+At Mechanical Rock, we achieve this by bringing UX together with DevOps, which enables lean development because the problem space is understood, solutions are tested with users prior to the build and iterations are based on data. Moreover, designers and developers work together with a shared context reducing misunderstanding and errors. Whereas, without UX, products often do not meet stakeholder or user needs ([Barnaby Dracup](https://www.devopsonline.co.uk/integrating-ux-with-devops/)).. In 2020, failed development projects cost $260 billion ([Roberto Torres](https://www.ciodive.com/news/poor-software-quality-report-2020/593015/)).
+
+Based on this, I believe value and subsqeuent competitive advantage is greatly enabled by understanding how to optimise the relationship between UX and DevOps.
+<br /><br />
+
+### 1. Discovery
+
+At Mechanical Rock before we start a piece of design work, we run in depth discovery sessions. Developers, designers and the clients are involved. In these sessions, we understand the problem space, dive deeper into the current state and user journey. A shared understanding is fostered as both designers and developers have the same context of the problem space.
+
+Without this session, it is too easy to make assumptions and be driven by the client leading to poor delivery. For example, on a recent project, the client was adamant that the users felt fearful of artifical intelligence and machine learning; however, our user research suggested otherwsie, which opened up new opportunities and allowed us to deliver solutions driven by user data. This is an example, of how UX provides visibility into the problem space and user needs which leads to strong delivery.
```

Comment:
```suggestion
Without this session, it is too easy to make assumptions and be driven by the client leading to poor delivery. For example, on a recent project, the client was adamant that the users felt fearful of artificial intelligence and machine learning; however, our user research suggested otherwise, which opened up new opportunities and allowed us to deliver solutions driven by user data. This is an example, of how UX provides visibility into the problem space and user needs which leads to strong delivery.
```